dotnet/src/Planners/Planners.Core/Handlebars/Extensions/SKParameterMetadataExtensions.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Planning.Handlebars.Models;
+
+#pragma warning disable IDE0130 // Namespace does not match folder structure
+namespace Microsoft.SemanticKernel.Planning.Handlebars.Extensions;
```

Comment:
Is this the best namespace?

Reply:
Removed `.Extensions`, just `namespace Microsoft.SemanticKernel.Planning.Handlebars` now